managed/services/checks/checks_test.go

```diff
@@ -810,7 +811,47 @@ func TestFillQueryPlaceholders(t *testing.T) {
 
 		query := "some query with service={{ .ServiceName }} and os={{ .OS }}"
 
-		_, err := fillQueryPlaceholders(query, target)
+		_, err := fillVMQueryPlaceholders(query, target)
 		require.EqualError(t, err, "failed to fill query placeholders: template: query:1:53: executing \"query\" at <.OS>: can't evaluate field OS in type struct { ServiceName string; NodeName string }")
 	})
 }
+
+func TestFillClickhouseQueryPlaceholders(t *testing.T) {
+	t.Parallel()
+
+	target := services.Target{
+		ServiceName: "test_service_name",
+		ServiceID:   "test_service_id",
+		NodeName:    "node_name",
+	}
+
+	t.Run("normal with placeholders", func(t *testing.T) {
+		t.Parallel()
+
+		query := "m_docs_scanned FROM metrics WHERE service_id={{.ServiceID}} AND period_start >= subtractHours(now(), 1) AND col1 < 10"
+		expected := "m_docs_scanned FROM metrics WHERE service_id=test_service_id AND period_start >= subtractHours(now(), 1) AND col1 < 10"
```

Comment:
I'm not a Clickhouse expert, but shouldn't the value `test_service_id` be quoted here?